src/net.cpp

```diff
@@ -2023,11 +2023,13 @@ void CConnman::ThreadOpenConnections(const std::vector<std::string> connect)
 
         int64_t nANow = GetAdjustedTime();
         int nTries = 0;
+        bool fDeleteAnchors = false;
```

Comment:
style nit
```suggestion
        bool delete_anchors_file{false};
```

Reply:
I prefer `delete_anchors_file`, didn't do it just to follow the same pattern of other vars in this file.

Reply:
Not a blocker but I believe it's preferred to use the current style guidelines for new code.

Reply:
Done! Thanks!